controllers/managedocs_controller.go

```diff
@@ -911,12 +911,18 @@ func (r *ManagedOCSReconciler) reconcileAlertRelabelConfigSecret() error {
 		}
 
 		alertRelabelConfig := []struct {
-			TargetLabel string `yaml:"target_label,omitempty"`
-			Replacement string `yaml:"replacement,omitempty"`
-		}{{
-			TargetLabel: "namespace",
-			Replacement: r.namespace,
-		}}
+			SourceLabels []string `yaml:"source_labels"`
+			TargetLabel  string   `yaml:"target_label,omitempty"`
+			Replacement  string   `yaml:"replacement,omitempty"`
+		}{
+			{
+				SourceLabels: []string{"namespace"},
+				TargetLabel:  "alertnamespace",
+			},
+			{
+				TargetLabel: "namespace",
+				Replacement: r.namespace,
+			}}
```

Comment:
```suggestion
			}
		}
```

Reply:
So the problem was for all the alerts that were raised from a different namespace, other than op-storage also have the same namespace i.e: `openshift-storage`.
Before this Relabel had just
```
TargetLabel: "namespace",
Replacement: r.namespace,
```
which was necessary so that our alertmanager can use the matchLabel of `namespace: openshift-storage`
Now what I have done this, just before relabelling the original namespace(alert original namespace) to openshift-storage(r.namespace), we are creating a new label with `alertnamespace` which can hold the value of original namespace and then we can relabel namespace.

Reply:
@akshayashu Can you add the details to the description of the PR? It would help for reference in the future.